server/MyMovieFlix/src/main/java/io/egen/movieflix/controller/UserController.java

```diff
@@ -0,0 +1,177 @@
+import io.egen.movieflix.entity.Title;
+import io.egen.movieflix.entity.UserTitleComment;
+import io.egen.movieflix.entity.UserTitleRating;
+import io.egen.movieflix.service.impl.TitleServiceImpl;
+import io.egen.movieflix.service.impl.UserServiceImpl;
+import io.egen.movieflix.utils.ApplicationLogger;
+import io.egen.movieflix.utils.ApplicationUtils;
+import io.egen.movieflix.utils.ConversionUtils;
+
+@RestController
+@RequestMapping(value = "/user")
+public class UserController {
+
+	@Autowired
+	TitleServiceImpl titleservice;
+
+	@Autowired
+	UserServiceImpl userService;
+
+	@Autowired
+	ApplicationLogger log;
+
+	@Autowired
+	ApplicationUtils utils;
+
+	@Autowired
+	ConversionUtils conversionUtils;
+
+	@RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+	public Account findAll() {
+		try {
+			return userService.find(1);
+		} catch (Exception e) {
+			log.error(e.getMessage());
+		}
+		return null;
+	}
+
+	/**
+	 * Generic function to handle view catalog and infinite scrolling
+	 * 
+	 * @param dto
+	 * @return
+	 */
+	@RequestMapping(value = "/catalog", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+	public TitlePaginationDTO findAllTitles(@RequestBody TitlePaginationDTO dto) {
+		try {
+			Integer fromIndex = dto.getFromIndex() != null ? (Integer.parseInt(dto.getFromIndex())) : 0;
+			Integer typeId = dto.getTitleTypeId() != null ? (Integer.parseInt(dto.getTitleTypeId())) : 0;
+			String year = (dto.getYear() != null || !dto.getYear().trim().equals("")) ? dto.getYear() : null;
+			String genre = (dto.getGenre() != null || !dto.getGenre().trim().equals("")) ? dto.getGenre() : null;
+			String title = (dto.getTitle() != null || !dto.getTitle().trim().equals("")) ? dto.getTitle() : null;
+			Boolean topRated = (dto.isViewTopRated()) ? true : null;
+			Integer searchCount = titleservice.findTotalCustomizedSearchTitleCount(typeId, year, genre, title,
+					topRated);
+			int fetchCount = (fromIndex + PAGINATION_SIZE) > searchCount ? (searchCount - fromIndex) : PAGINATION_SIZE;
+			List<Title> newTitleList = titleservice.fetchAllTitle(fromIndex, fetchCount);
+			List<TitleDTO> titleResult = new ArrayList<>();
+			if (newTitleList != null && !newTitleList.isEmpty()) {
+				for (Title entity : newTitleList) {
+					titleResult.add(conversionUtils.getTitleDTO(entity));
+				}
+			}
+			dto.setTitleList(titleResult);
+			dto.setFromIndex(String.valueOf(fromIndex + fetchCount));
+			return dto;
+		} catch (Exception e) {
+			log.error(e.getMessage());
+		}
+		return null;
+	}
+
+	/**
+	 * Fetch a title for user to comment, rate or view
+	 * 
+	 * @param titleId
+	 * @return
+	 */
+	@RequestMapping(value = "/title/{id}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+	public UserTitleUpdateDTO fetchTitleForViewing(@PathVariable("id") String titleId, HttpServletRequest request) {
```

Comment:
Is a controller for `user` entity returning a response with `title`?
Ideally `UserController` should be playing with `User` entities only.